amundsen_application/static/js/components/ProfilePage/index.tsx

```diff
@@ -204,4 +224,4 @@ export const mapDispatchToProps = (dispatch) => {
   return bindActionCreators({ getUserById, getUserOwn, getUserRead, getBookmarksForUser }, dispatch);
 };
 
-export default connect<StateFromProps, DispatchFromProps>(mapStateToProps, mapDispatchToProps)(ProfilePage);
+export default connect<StateFromProps, DispatchFromProps>(mapStateToProps, mapDispatchToProps)(withRouter(ProfilePage));
```

Comment:
what is `withRouter` doing here?

Reply:
The specific case where it's needed is when you link a profile page while the profile page is already loaded. E.g. routing from `/user/tfeng@lyft.com` -> `/user/dwon@lyft.com` wasn't loading. `withRouter` is a `react-router` utility allows us to read updates when the route changes.